capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -12,6 +12,31 @@ const DOWN_KEY = 'ArrowDown';
 const ENTER_KEY = 'Enter';
 let selectedSuggestionPosition = -1;
 
+/**
+ * Gets suggestions to display to the users
+ */
+function getSuggestions() {
+  const fetchStr = '/search?query=' + SEARCH_INPUT.value;
+  fetch(fetchStr).then((response) => response.json()).then((suggestions) => {
+    giveSuggestions(suggestions);
```

Comment:
same sugegstion about method naming [in part 1](https://github.com/googleinterns/univiz/pull/50/files/1e428a8aacd5b996e3fe6fcb87150648e123ba1b#r478783432).

Reply:
alrighty, I'll go through and try to make the naming scheme more sensible! thanks!